.github/workflows/tests.yml

```diff
@@ -186,8 +251,12 @@ jobs:
           path: bottles
 
       - run: brew test-bot --only-cleanup-after
-        if: always()
+        if: always() && runner.os == 'macOS'
 
       - name: Post Cleanup
         if: always()
         run: rm -rvf bottles
+
+      - name: Remove container
+        if: always() && runner.os == 'Linux'
+        run: docker rm -f linuxbrew
```

Comment:
Could consider combining all of these into a single cleanup step.

Reply:
Good call. Will try later.

Reply:
Thanks!